scripts/local_config.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python3
+
+"""
+This code is imported from other scripts to read paths from the
+configuration file.
+"""
+
+import os
+import sys
+
+
+def read_config(params):
+    # Get absolute path of the repository's root from the script location
+    root_folder = os.path.abspath(os.path.join(
+        os.path.dirname(__file__), os.pardir))
+    config_file = os.path.join(root_folder, 'config', 'config')
+
+    # Abort if config file is missing
+    if not os.path.exists(config_file):
+        print('ERROR: config file is missing')
+        sys.exit(1)
+
+    # Read all available paths in the config file
+    paths = {}
+    with open(config_file, 'r') as cfg:
+        lines = cfg.readlines()
```

Comment:
As I'm nitpicking below anyway, TIL: iterating over a file is the same as `readlines`.

I pondered putting all of this into one big generator, but my brain chickened out. Consider just putting it into one loop, though? And `continue` on empty/comment lines?